src/etl/sars-cov-2/steps/add-vaccination-data-to-estimate-step.ts

```diff
@@ -36,29 +126,34 @@ interface AddVaccinationDataToEstimateStepOutput {
 export const addVaccinationDataToEstimateStep = (
   input: AddVaccinationDataToEstimateStepInput
 ): AddVaccinationDataToEstimateStepOutput => {
+  const acceptableDayDifferenceRange = 31;
```

Comment:
I don't know what the magic number is but basically if your country is reporting at least monthly we're good.